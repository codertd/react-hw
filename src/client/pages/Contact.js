import React from 'react';

class Contact extends React.PureComponent {
  render() {
    return <h1>Contact</h1>;
  }
}

export default Contact;
