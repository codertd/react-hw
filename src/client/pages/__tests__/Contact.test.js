// __tests__/Contact.test.js

import React from 'react';
import {
  render
} from '@testing-library/react';
import Contact from '../Contact';

it('renders', () => {
  const { asFragment } = render(<Contact text="Hello!" />);
  expect(asFragment()).toMatchSnapshot();
});
