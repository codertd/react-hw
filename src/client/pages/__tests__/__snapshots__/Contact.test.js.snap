// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <h1>
    Contact
  </h1>
</DocumentFragment>
`;
